### PR TITLE
Create new Analytics for every spatialAnalytics tool

### PR DESCRIPTION
**Pull request by hobinjk-ptc**
Simplifies AnalyticsMobile implementation
Fixes various state issues with focus/blur/open/close
Enables reuse of previously loaded data (no more reloading the entire mercedes dataset every time you open the tool)

### CODE REVIEW

File: src/analytics/index.js [hunk 1]
```diff
@@ -4,74 +4,133 @@ import {Analytics} from './analytics.js'
...
+     */
+    function makeAnalytics(frame) {
+        if (realityEditor.device.environment.isDesktop()) {
+            return new Analytics(frame);
+        } else {
+            return new AnalyticsMobile(frame);
         }
+    }
+
+    const noneFrame = 'none';
```
Comment (dangond-ptc) [inline src/analytics/index.js:19]: Is there a reason to use this instead of `null`?

**Review by dangond-ptc — APPROVED** [review]
Aside from one small nit, looks good to me! Really glad to see timelines and HPAs more strongly managed by a single frame under one Analytics object, makes it much easier to reason about what's going on.

File: src/analytics/index.js (hunk 1 above)
Comment (hobinjk-ptc) [reply src/analytics/index.js:19]: The use of a string here is entirely because it felt weird to index by null

Comment (dangond-ptc) [reply src/analytics/index.js:19]: Fair enough

Comment (hobinjk-ptc) [reply src/analytics/index.js:19]: If you prefer, I can make up something about how using null would trigger an engine-level deoptimization of the underlying javascript map

**Pull request merged**